src/sender.cpp

```diff
@@ -339,6 +349,8 @@ bool SenderPrivate::sendMail(const MimeMessage &email)
     }
     qCDebug(SIMPLEMAIL_SENDER) << "Mail sent";
 
+    mailQueueId = QString(responseText).trimmed().section("queued as", 1, 1, QString::SectionCaseInsensitiveSeps).trimmed();
```

Comment:
Please trim on the responseText bytearray then avoid UTF8 conversion by using fromLatin1(), static text should always be either QLating1String or QStringLiteral, but since section is a slow method please move this code to the call to mailQueueId, so that this is only payed by those who want the id.

Reply:
I can switch to using indexOf and mid() but imho we are dealing with network communication in this class. This string operation on a small string is so insignificant that I would favor a clear interface and a stable implementation over the very small performance gain.

Otherwise you have to store the response text because it should still be available when calling quit

Reply:
Due to the fact that not every mail server might use the same format for stating the mail queue ID, I switched to storing the response and parsing it when querying mailQueueId(). In case the format is not known the entire last response is returned.